libs/transactions/withdrawal.ts

```diff
@@ -0,0 +1,61 @@
+import type Web3 from "web3";
+import path from "path";
+import { parentLogger } from "../utils";
+import { submitTransaction } from "./helpers/submit";
+import type { Client } from "../client";
+import type { NightfallZkpKeys } from "../nightfall/types";
+import type { TransactionReceipt } from "web3-core";
+
+const logger = parentLogger.child({
+  name: path.relative(process.cwd(), __filename),
+});
+
+export async function createAndSubmitWithdrawal(
```

Comment:
IMO we should change the name for handleWithdrawal because is what this function do. And rename submitTransaction for createAndSubmitTransaction that is what this function do. What do you think?

Reply:
As discussed, all transactions on-chain receive an `unsignedTx`, which is passed later to `submitTransaction`. So we'll keep this naming pattern.